gamemodes/terrortown/gamemode/client/cl_shopeditor.lua

```diff
@@ -57,9 +57,9 @@ function ShopEditor.GetEquipmentForRoleAll()
 		local name = WEPS.GetClass(v)
 
 		if name
-		and not v.Doublicated
-		and not string.match(name, "base")
-		and not eject[name]
+			and not v.Doublicated
+			and not string.match(name, "base")
+			and not eject[name]
```

Comment:
we never did things like this. Probably something we have ti normalize regarding CS